CONTRIBUTING.md

```diff
@@ -34,9 +34,7 @@ Issues that need confirmation will have the **confirm** label or be unlabeled an
 - Create unit tests to demonstrate the issue
 - Test issues and provide feedback
 
-As of version 3.10, the NUnit and NUnitLite NuGet packages support **debugger source-stepping** for each binary from the repository. Debuggers can step into the source code, set breakpoints, watch variables, etc. If you’re getting ready to report a bug in NUnit, figuring out how to create a minimal repro is much easier since you aren’t dealing with a black box!
-
-To easily drop into NUnit code from your project any time, [follow these steps](https://github.com/nunit/docs/wiki/Debugger-Source-Stepping).
+If you’re getting ready to report a bug in NUnit, figuring out how to create a minimal repro is easier if you temporarily disable the debugger’ [Just My Code](https://docs.microsoft.com/en-us/visualstudio/debugger/just-my-code) setting. This allows you to step into NUnit's source code, set breakpoints, watch variables, etc.
```

Comment:
Being a non-native English speaker I'm unsure whether "debugger’" should be "debugger’s"? Otherwise I think it looks good

Reply:
This reads fine to me. I think it should just be debugger.

Reply:
Ugh, I did mean to type `debugger’s`.